service/auth.go

```diff
@@ -0,0 +1,67 @@
+package service
```

Comment:
yes, that would be easier if all AuthN/AuthZ stuff belongs to the `auth` pkg, indeed

Reply:
I see pkg name `token` is meaningful for the content it has atm.  I am not sure is in future `token` pkg get something which is unrelated to `auth` at all in that case touching `token` pkg would not be right.

In case we want to do this refactoring, I still want to see `token` as sub-package in itself.
```
auth
   -> token
           (all existing token pkg file)
   -> service
          (this new auth.go file but named service.go)
```
With this input .. I will leave final call for @alexeykazakov

Reply:
Well.. `token` is kinda sub area of `auth` anyway. Can't imagine anything which we would like to keep in `token` which is unrelated to `auth` at all right now. But if we have it in the future we can always introduce a new `token` package outside of `auth`.
For now I would have `auth` only.
While I would be fine with having:
```
auth
   -> token
            (all existing token pkg files and jwk subpackage)
   -> service.go (no need for service subpackage)
```
But IMO
```
auth
   -> 
        (all existing token pkg files and jwk subpackage)
        service.go (no need for service subpackage)
```
is even simpler. But I'm fine with both options.

Reply:
Done in https://github.com/fabric8-services/fabric8-common/pull/62/commits/fb124aa132659c2325a6b9aaf30b7f88c3915f6d